CodingChallenges/CC_046_Asteroids/Processing/CC_046_Asteroids/CC_046_Asteroids.pde

```diff
@@ -0,0 +1,79 @@
+// Daniel Shiffman
+// http://codingtra.in
+// http://patreon.com/codingtrain
+// Code for: https://youtu.be/hacZU523FyM
+// Processing transcription: Chuck England
+
+import java.util.*;
+
+Ship ship;
+List<Asteroid> asteroids = new ArrayList<Asteroid>();
```

Comment:
Same question here re: `ArrayList`

Reply:
Here you can see the interface pull-off more clearly. I depend on the List<> interface I pulled off the ArrayList<> implementation.